Projects/Domain/FileDomain/Sources/DataSource/RemoteFileDataSourceImpl.swift

```diff
@@ -0,0 +1,11 @@
+import FileDomainInterface
+import Foundation
+import BaseDomain
+
+final class RemoteFileDataSourceImpl: BaseRemoteDataSource<FileEndpoint>, RemoteFileDataSource {
+    func dreamBookUpload(file: Data, fileName: String) async throws -> String {
+        try await request(
+            .dreamBookUpload(file: file, fileName: fileName),
+            dto: DreamBookUploadResponseDTO.self).fileURL
+    }
```

Comment:
```suggestion
        try await request(
            .dreamBookUpload(file: file, fileName: fileName),
            dto: DreamBookUploadResponseDTO.self
        )
        .fileURL
    }
```
줄바꿈할거라면 이렇게 하는게 스타일 가이드에 맞을거같아요

Reply:
오 알겠습니다